TYPO3.Neos.Kickstarter/Resources/Private/Generator/TypoScript/Root.ts2

```diff
@@ -1,37 +1,41 @@
 /**
  * Root TypoScript template for the {siteName} site
+ *
+ * For more details see: http://neos.readthedocs.io/en/stable/HowTos/SelectingPageTemplate.html#using-a-defaultpage-prototype
  */
-page = TYPO3.Neos:Page {
-	head {
-		stylesheets.site = TYPO3.TypoScript:Template {
-			templatePath = 'resource://{packageKey}/Private/Templates/Page/Default.html'
-			sectionName = 'stylesheets'
-		}
 
-		javascripts.site = TYPO3.TypoScript:Template {
-			templatePath = 'resource://{packageKey}/Private/Templates/Page/Default.html'
-			sectionName = 'headScripts'
-		}
-	}
+root {
+    default {
+        type = ${q(node).property('_nodeType') + '.Document'}
+        renderPath >
+    }
+}
 
-	body {
-		templatePath = 'resource://{packageKey}/Private/Templates/Page/Default.html'
-		sectionName = 'body'
-		parts {
-			menu = TYPO3.Neos:Menu
-		}
+prototype(Neos:DefaultPage) < prototype(Page) {
```

Comment:
What is `Neos` supposed to be? We generally always use packageKeys there so IMHO

`prototype({packageKey}:DefaultPage) < prototype(Page) {`